packages/core/lib/debug/printer.js

```diff
@@ -172,13 +172,48 @@ class DebugPrinter {
     const colorizedLines = splitLines(colorizedSource);
 
     this.config.logger.log("");
+    // We create printoutRange with range.lines as initial value for printing.
+    let printoutRange = {
```

Comment:
Why not just do `let printoutRange = range.lines`?  You're not going to be altering its components, so there isn't a need to do a clone like this.